src/pages/AboutUs.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import NavBar from "../components/navigation/NavBar";
+import { makeStyles, Typography, Link } from "@material-ui/core";
+
+import backgroundImage from  "../media/about_us_background.png"
```

Comment:
that image is quite heavy to load - if its the same loaded everywhere, we would need to cache it to enhance loading speed

Reply:
Are we using this image somewhere else? Can we load it in the palette instead then?